examples/cellar/gen/http/storage/server/encode_decode.go

```diff
@@ -193,6 +193,49 @@ func DecodeRateRequest(mux goahttp.Muxer, decoder func(*http.Request) goahttp.De
 	}
 }
 
+// EncodeMultiAddResponse returns an encoder for responses returned by the
+// storage multi_add endpoint.
+func EncodeMultiAddResponse(encoder func(context.Context, http.ResponseWriter) goahttp.Encoder) func(context.Context, http.ResponseWriter, interface{}) error {
+	return func(ctx context.Context, w http.ResponseWriter, v interface{}) error {
+		res := v.([]string)
+		enc := encoder(ctx, w)
+		body := res
+		w.WriteHeader(http.StatusOK)
+		return enc.Encode(body)
+	}
+}
+
+// NewStorageMultiAddDecoder returns a decoder to decode the multipart request
+// for the "storage" service "multi_add" endpoint.
+func NewStorageMultiAddDecoder(storageMultiAddDecoderFn StorageMultiAddDecoderFunc) func(r *http.Request) goahttp.Decoder {
+	return func(r *http.Request) goahttp.Decoder {
+		return goahttp.EncodingFunc(func(v interface{}) error {
+			mr, err := r.MultipartReader()
+			if err != nil {
+				return err
+			}
+			p := v.(*[]*storage.Bottle)
```

Comment:
Same as above

Reply:
I think here we have to pass a pointer so that the underlying decoder func can populate this variable.